tls/s2n_x509_validator.c

```diff
@@ -648,6 +648,41 @@ static S2N_RESULT s2n_x509_validator_disable_time_validation(struct s2n_connecti
     return S2N_RESULT_OK;
 }
 
+int s2n_no_op_verify_custom_crit_oids_cb(X509_STORE_CTX *ctx, X509 *x509, STACK_OF(ASN1_OBJECT) *oids)
+{
+    return 1;
+}
+
+static S2N_RESULT s2n_x509_validator_add_custom_extensions(struct s2n_x509_validator *validator, struct s2n_connection *conn)
+{
+    RESULT_ENSURE_REF(validator);
```

Comment:
nit - the `store_ctx` is given to the libcrypto, so probably worth checking it first.
```suggestion
    RESULT_ENSURE_REF(validator);
    RESULT_ENSURE_REF(validator->store_ctx);
```